substrate/frame/collective/src/benchmarking.rs

```diff
@@ -57,7 +57,7 @@ benchmarks_instance_pallet! {
 
 		Collective::<T, I>::set_members(
 			SystemOrigin::Root.into(),
-			old_members.clone(),
+			old_members.clone().try_into().unwrap(),
```

Comment:
Can you init all these as `bounded_vec` so you don't need to `try_into`?

Reply:
the alternative is to have many `members.try_push(member).unwrap()`